recipes/idna/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "idna" %}
+{% set version = "2.1" %}
+{% set hash_type = "sha256" %}
+{% set hash = "ed36f281aebf3cd0797f163bb165d84c31507cedd15928b095b1675e2d04c676" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - idna
+
+about:
+  home: https://github.com/kjd/idna
+  license: BSD Like
```

Comment:
Yep, I looked at the license file and would generally agree. It was a BSD 3-Clause license with an additional (not advertising) clause similar in form to the HDF5 license. Though there are parts under other licenses like PSF 2 and some BSD like license for Unicode stuff. We could list those out, but "BSD Like" seems sufficient to cover those generally (at least to me).

We could ask others if they have a preference.